src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,54 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event to be done at a specified date.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Creates a new Event with the given description and date.
+     * @param description is the description of the event.
+     * @param at is the event date.
+     */
+    public Event(String description, String at) {
+        super(description);
+        this.at = LocalDate.parse(at);
+    }
+
+    /**
+     * Gives an Event task from the full task description in the file.
+     * @param loadTask is the full task description in the file.
+     * @return an Event task.
+     */
+    public static Event load(String loadTask) {
+        String[] splitTask = loadTask.split(" \\| ", 4);
+        Event event = new Event(splitTask[2], splitTask[3]);
+        if (splitTask[1].equals("1")) {
+            event.markAsDone();
+        }
+        return event;
+    }
```

Comment:
I think its a bit strange to have a load method inside a Task class? Maybe you can consider having a SaveAndLoadManager to handle all the saving and loading of tasks? Just a suggestion hehe